### PR TITLE
formatting(docstrings): fix common errors

### PR DESCRIPTION
**Pull request by NripeshN**

### CODE REVIEW

File: ivy/functional/ivy/experimental/linear_algebra.py [hunk 1]
```diff
@@ -303,10 +304,12 @@ def matrix_exp(
 
     Examples
     --------
-        >>> x = ivy.array([[[1., 0.],
+
+    >>> x = ivy.array([[[1., 0.],
                             [0., 1.]],
                             [[2., 0.],
                             [0., 2.]]])
+
```
Comment (vedpatwardhan) [inline ivy/functional/ivy/experimental/linear_algebra.py:312]: we'd probably need to also format the lines that follow to fix the indentation

**Review by vedpatwardhan — CHANGES_REQUESTED** [review]
I've primarily gone through all framework files changed, and except one fix everything looks good with the framework files. Thanks @NripeshN 😄

File: ivy/functional/ivy/device.py [hunk 2]
```diff
@@ -656,14 +668,19 @@ def dev_util(device: Union[ivy.Device, ivy.NativeDevice], /) -> float:
 
     Example
     -------
+
     >>> ivy.dev_util('cpu')
     13.4
+
     >>> ivy.dev_util('gpu:0')
     7.8
+
```
Comment (lucasalavapena) [inline ivy/functional/ivy/device.py:677]: What is the rationale behind this newline?

File: ivy/data_classes/array/elementwise.py [hunk 3]
```diff
@@ -1815,6 +1843,7 @@ def maximum(
         Examples
         --------
         With :class:`ivy.Array` inputs:
+
```
Comment (lucasalavapena) [inline ivy/data_classes/array/elementwise.py:1846]: Is this intentional? If so why? I thought you were adding the newline to separate the title from the body or is it to have the line before the codeblock (I seemed to have seen comments on both on your other PR)

File: ivy/data_classes/array/activations.py [hunk 4]
```diff
@@ -40,6 +40,7 @@ def relu(
 
         Examples
         --------
+
```
Comment (lucasalavapena) [inline ivy/data_classes/array/activations.py:43]: Why are we adding a newline here? Why only after examples, but not returns, parameters? You failing the format checks :sweat_smile:  D412: No blank lines allowed between a section header and its content ('Examples')

**Review by lucasalavapena — CHANGES_REQUESTED** [review]
Thanks for the PR @NripeshN , I left some comments/general questions, make sure to check the pydocstyle linting errors in the CI too!

**Review by KareemMAX — CHANGES_REQUESTED** [review]
As @lucasalavapena mentioned, adding new lines after examples would trigger `D412` on `pydocstyle`. Maybe remove that so we can get a more clearer picture of what changes does happen

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by vedpatwardhan** [comment]
Hey @NripeshN @KareemMAX, is this PR still being worked on? Thanks!

**Pull request closed without merging**